packages/slice-machine/components/Simulator/components/SetupModal/CodeBlock.tsx

```diff
@@ -5,9 +5,16 @@ import { MdCheck, MdContentCopy } from "react-icons/md";
 
 import CodeBlock, { Language } from "../../../CodeBlock";
 import { IconType } from "react-icons";
+import VersionBadgeList from "./VersionBadgeList";
+
+type Code = {
+  text: string;
+  version: string;
+};
 
 const CodeBlockWithCopy: React.FC<{
-  children: string;
+  children?: string;
```

Comment:
I'm not a fan of having two possible source. I'd rather have `children: string | Code[]` and deal with it in the component. It feels cleaner, makes the interface cleaner and easier to understand and user errors less likely (here could be called with neither children nor codelist)

Reply:
Yeah me either. But I'm not a fan of children being anything but a node. I'll find something else

Reply:
Would it be ok for you if I removed `children` and only kept `code`, of type `Code | Code[]` ?

Reply:
https://github.com/prismicio/slice-machine/pull/794/commits/88613f6af5ae95fc842bdbe565e46716e77ee802#diff-9d2f70d013cc7c10e6d41982d7fdd59904c6c12627482ffd9a9ae8546146024bR16

I updated, LMK!